include/openenclave/internal/sgx/plugin.h

```diff
@@ -33,11 +33,11 @@ typedef enum _sgx_evidence_format_type_t
  * @param[in] format_id Pointer to the evidence format ID requested.
  * @param[in] report_body Pointer to a buffer with raw SGX quote / report.
  * @param[in] report_body_size Size of the report_body buffer.
- * @param[in] sgx_endorsements Pointer to the endorsements buffer.
- * @param[in] custom_claims_buffer The data in a flat buffer to be packaged as a
- * claim of ID OE_CLAIM_CUSTOM_CLAIMS_BUFFER when format_type has the right
- * value.
+ * @param[in] custom_claims_buffer If not NULL, it holds the data in a flat
+ * buffer to be packaged as a claim of ID OE_CLAIM_CUSTOM_CLAIMS_BUFFER when
+ * format_type has the right value.
  * @param[in] custom_claims_buffer_size The size of the custom_claims buffer.
```

Comment:
Shouldn't the size param also say "If not NULL, ..."?

Reply:
`custom_claims_buffer_size` is of type `size_t`, so it can't be NULL.